client/plots/matrix.controls.js

```diff
@@ -399,7 +399,10 @@ export class MatrixControls {
 								width: 50,
 								align: 'center',
 								chartType: 'matrix',
-								inputs: [{ label: 'N/A' }, { settingsKey: 'rowh', min: 8, max: 30, step: 1 }]
+								inputs: [{ label: 'N/A' }, { settingsKey: 'rowh', min: 8, max: 30, step: 1 }],
+								getDisplayStyle(plot) {
```

Comment:
@congyu-lu User's may prefer a short row height to be able to view more rows without having to scroll.

Reply:
changing row height now will make the left dendrogram misaligned. should we disable it for now before it's fixed?

Reply:
@congyu-lu we should try to fix this in a new branch.